### PR TITLE
Added Identification document section to patient registration form

### PR DESCRIPTION
**Pull request by MiguelAHPpih**

### CODE REVIEW

**Comment by brandones** [comment]
Please resolve the conflict

**Comment by brandones** [comment]
@MiguelAHPpih I imagine that, per our conversation today, we're going to be re-doing this using patient identifier types? We can merge this anyway if you'd like it merged now for whatever reason. Or we can close.

File: api/src/main/java/org/openmrs/module/mirebalais/apploader/apps/patientregistration/SectionsPeru.java [hunk 1]
```diff
@@ -1,6 +1,9 @@
 package org.openmrs.module.mirebalais.apploader.apps.patientregistration;
 
 import org.openmrs.module.pihcore.config.Config;
+import org.openmrs.module.pihcore.config.registration.SocialConfigDescriptor;
+import org.openmrs.module.pihcore.metadata.mexico.MexicoPatientIdentifierTypes;
```
Comment (brandones) [inline api/src/main/java/org/openmrs/module/mirebalais/apploader/apps/patientregistration/SectionsPeru.java:5]:
```suggestion
```

File: api/src/main/java/org/openmrs/module/mirebalais/apploader/apps/patientregistration/SectionsPeru.java [hunk 2]
```diff
@@ -18,12 +20,56 @@ public SectionsPeru(Config config) {
...
+        return s;
+    }
+    
+    private Question getDNIdocument() {
+        Question q = new Question();
+        q.setId("national-id");
+        q.setLegend("zl.registration.patient.documenttype.dni.label");
+        q.setHeader("zl.registration.patient.documenttype.dni.label");
+
+        Field f=new Field();
```
Comment (brandones) [inline api/src/main/java/org/openmrs/module/mirebalais/apploader/apps/patientregistration/SectionsPeru.java:45]:
```suggestion
        Field f = new Field();
```

File: api/src/main/java/org/openmrs/module/mirebalais/apploader/apps/patientregistration/SectionsPeru.java [hunk 3]
```diff
@@ -18,12 +20,56 @@ public SectionsPeru(Config config) {
...
+
+        return q;
+    }
+    private Question getPassportdocument() {
+        Question q = new Question();
+        q.setId("passport-id");
+        q.setLegend("zl.registration.patient.documenttype.passport.label");
+        q.setHeader("zl.registration.patient.documenttype.passport.label");
+
+        Field f=new Field();
```
Comment (brandones) [inline api/src/main/java/org/openmrs/module/mirebalais/apploader/apps/patientregistration/SectionsPeru.java:61]:
```suggestion
        Field f = new Field();
```

**Review by brandones — APPROVED** [review]
@MiguelAHPpih This seems to be working on your end? The code looks good.

**Comment by brandones** [comment]
I guess I'll merge this? Not sure what's going on here.

**Comment by mogoodrich** [comment]
Looks like this is dependent on some changes to PIH Core as well?  It's throwing a compile error in build: http://bamboo.pih-emr.org:8085/browse/MIREBALAIS-INSTALL-12116/log

No worries, but I'm going to revert this for now until we get the PR for PIH Core.

**Pull request merged**